Source/WorkTimeTracker/ViewModels/MainViewModel.cs

```diff
@@ -99,6 +110,13 @@ internal async Task LoadWorkTimes()
             WorkTimes.Replace(_workTimes);
         }
 
+        internal async Task LoadSettings()
+        {
+            _settings = await _settingsStorage.Load();
+
+            SelectedFilter = Filters.FirstOrDefault(x => x.Filter == _settings.LastSelectedFilter);
```

Comment:
What happens when the settings don't contain the filter property?

Reply:
Filter is a not nullable enumeration and "None" is the filter value you'll get when a new instance of Settings is returned by the Load method.